scripts/start.js

```diff
@@ -3,22 +3,24 @@ const { spawn } = require( 'child_process' );
 const fs = require( 'fs' );
 const path = require( 'path' );
 const onExit = require( 'signal-exit' );
+const overrideConfigCache = require( './override-start' );
 
 const {
-  choosePort,
-  createCompiler,
-  prepareProxy,
-  prepareUrls,
+	choosePort,
+	createCompiler,
+	prepareProxy,
+	prepareUrls,
```

Comment:
This project exists in a middle-ground between the React project's style and WordPress's; I'd ask for _purely_ stylistic comments to be held until a later PR, I recognize there is inconsistency here :)